### PR TITLE
Rclobus/typos

### PR DESCRIPTION
**Pull request by rclobus**
Minor corrections to the manual

### CODE REVIEW

**Comment by rclobus** [comment]
Please squash them into one commit when merging, the details are IMO not very important.

**Comment by rclobus** [comment]
The changes have been verified in the HTML version of the asciidoc files.

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -270,8 +270,7 @@ a2enmod proxy_wstunnel
 a2enmod rewrite
 --------------------------------------------------------------------------------
 
-For a basic setup, you can copy *openqa.conf.template* to *openqa.conf* and modify the `ServerName` if required
-setting. This will direct all HTTP traffic to openQA.
+For a basic setup, you can copy *openqa.conf.template* to *openqa.conf* and modify the `ServerName` setting if required. This will direct all HTTP traffic to openQA.
```
Comment (Martchus) [inline docs/Installing.asciidoc:273]: It would be nice to keep the wrapping.

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -797,8 +796,8 @@ directories prior registering with openQA instance:
 4. `/var/lib/openqa`
 5. fail if none of above is available
 
-Once worker registers to openQA instance it checks for available job and starts
-accepting websockets commands. Worker accepts jobs as they will come in, there
+Once a worker registers to an openQA instance it checks for available jobs and starts
+accepting websockets commands. Workers accept jobs as they will come in, there
 is no priority, or other ordering, support at the moment.
```
Comment (Martchus) [inline docs/Installing.asciidoc:801]: This whole section is actually completely wrong. A worker does never check for available jobs. It is possible to prioritize jobs.

So this should be: "Once a worker registers to an openQA instance, a scheduled job (of matching worker class) will be assigned to it. Dependencies between jobs will be considered for ordering the job assignment. Generally, jobs with higher priority (lower priority value) will be assigned first." - Although I'm wondering whether we need to go into that much detail here anyways.

**Review by Martchus — COMMENTED** [review]
It would be nice to start commit message still with a capital letter (*after* the  `documentation: ` prefix). Additionally, I'd prefer "Improve grammar" over just "grammar".

**Comment by Martchus** [comment]
I've just seen you want us to squash the commits. I can do that; so forget the remarks about the commit message. I've only added "not-ready" to prevent mergify from merging this so I can squash the commits instead.

**Comment by okurz** [comment]
Still the individual comments need to be addressed, right? At least I see a need for rewrapping.

File: docs/Installing.asciidoc (hunk 2 above)
Comment (rclobus) [reply docs/Installing.asciidoc]: I've simplified it to:
"Once a worker registers to an openQA instance, scheduled jobs (of matching
worker class) can be assigned to it. Dependencies between jobs will be
considered for ordering the job assignment."

File: docs/Installing.asciidoc (hunk 1 above)
Comment (rclobus) [reply docs/Installing.asciidoc]: Done.

**Comment by rclobus** [comment]
I've squashed the commits. Wrapping has been applied to match the original line width. A simplified statement what happens when a worker is registered has been entered.

**Review by Martchus — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**